thrift/thrift0.13/src/main/java/com/linecorp/armeria/server/thrift/ThriftServiceEntry.java

```diff
@@ -73,6 +74,16 @@ public Set<Class<?>> interfaces() {
         return metadata.interfaces();
     }
 
+    /**
+     * Returns the associated thrift method name in `.thrift` from Thrift function.
+     * @return the associated thrift method name in `.thrift` function.
```

Comment:
Let's remove the `@return ...` sentence because it doesn't have much meaning from the sentence above.
```suggestion
     * Returns the associated thrift method name in the Thrift IDL.
```